docs/sources/next/using-k6/k6-options/reference.md

```diff
@@ -1476,7 +1476,13 @@ The possible keys with their meanings and default values:
 | proto               | the protocol to use when connecting with the traces backend                                                            | `grpc`                  |
 | header.`headerName` | adds an additional HTTP header with the provided header name and value to each HTTP request made to the traces backend | N/A                     |
 
-## Upload Only
+## Upload Only (deprecated)
+
+{{< admonition type="caution" >}}
+
+The "Upload Only" option is deprecated and will be removed in a future release. Please use the `k6 cloud upload` command instead.
```

Comment:
```suggestion
## Upload only (deprecated)

{{< admonition type="caution" >}}

The "Upload only" option is deprecated and will be removed in a future release. Use the `k6 cloud upload` command instead.
```

Reply:
I think it would be worth it indeed, especially as some of those have very specific usage or behaviors that we currently don't really document here.